tests/msc2716_test.go

```diff
@@ -751,22 +886,64 @@ func createMessagesInRoom(t *testing.T, c *client.CSAPI, roomID string, count in
 	return eventIDs
 }
 
-var chunkCount int64 = 0
+func createInviteStateEventsForBackfillRequest(
+	invitedByUserID string,
+	virtualUserIDs []string,
+	insertTime time.Time,
+) []map[string]interface{} {
+	// Timestamp in milliseconds
+	insertOriginServerTs := uint64(insertTime.UnixNano() / int64(time.Millisecond))
 
-func batchSendHistoricalMessages(
-	t *testing.T,
-	c *client.CSAPI,
+	state_evs := make([]map[string]interface{}, len(virtualUserIDs))
```

Comment:
Please don't use `_` in variable names. Go uses camel case.

Reply:
🙇 Python -> Go copy-paste. Updated 👍